### ISSUE TITLE
Doesn't get list of movies from Radarr

### ISSUE THREAD

**Issue by BodingClockchian** [morpheus65535/bazarr#162]
**Describe the bug**
`29/09/2018 13:01:17|ERROR|Job "Update movies list from Radarr (trigger: interval[0:05:00], next run at: 2018-09-29 13:06:16 CEST)" raised an exception|'Traceback (most recent call last):\n  File "libs/apscheduler\\executors\\base.py", line 125, in run_job\n    retval = job.func(*job.args, **job.kwargs)\n  File "C:\\bazarr\\get_movies.py", line 89, in update_movies\n    c.execute(\'\'\'UPDATE table_movies SET title = ?, path = ?, tmdbId = ?, radarrId = ?, overview = ?, poster = ?, fanart = ?, `audio_language` = ?, sceneName = ?, monitored = ? WHERE tmdbid = ?\'\'\', (movie["title"],movie["path"] + separator + movie[\'movieFile\'][\'relativePath\'],movie["tmdbId"],movie["id"],overview,poster,fanart,profile_id_to_language(movie[\'qualityProfileId\']),sceneName,unicode(bool(movie[\'monitored\'])),movie["tmdbId"]))\nKeyError: \'movieFile\''|`

**To Reproduce**
Manual Radarr sync

**Expected behavior**
Get the movies from Radarr

**Screenshots**
Nope

**Software (please complete the following information):**
 - Bazarr Ver. 0.6.4
 - Radarr version 0.2.0.1120
 - OS: Windows 7

Currently only using Radarr

**Comment by BodingClockchian** [comment]
```
libs/backports\configparser\__init__.py:1238: DeprecationWarning: You passed a b
ytestring. Implicitly decoding as UTF-8 string. This will not work on Python 3.
Please switch to using Unicode strings across the board.
  _, option, value = self._validate_value_types(option=option, value=value)
```
The console is outputting the text above sometimes.
I managed to get Bazarr working with Radarr but somehow its broken rn

**Comment by morpheus65535** [comment]
The first one will be fixed in 0.6.5.

@halali Could you take a look at this warning? Thanks.

**Comment by halali** [comment]
Warning should be fixed in 0.6.5.

**Comment by BodingClockchian** [comment]
Confirm. This issue is fixed in 0.6.5
